indexer/config.example.toml

```diff
@@ -0,0 +1,27 @@
+[Monitoring]
+Enabled = true
+Type = "beholder"
+
+[Monitoring.Beholder]
+InsecureConnection = true
+OtelExporterHTTPEndpoint = "otel-collector:4318"
+LogStreamingEnabled = true
+MetricReaderInterval = 5
+TraceSampleRatio = 1.0
+TraceBatchTimeout = 10
+
+[Scanner]
+ScanInterval = 1
+
+[Discovery]
+Type = "static"
```

Comment:
Why do you need the type here? The subsequent `Discovery.Static.*` headers include it already.